web/src/main/java/org/mskcc/oncotree/api/GlobalControllerExceptionHandler.java

```diff
@@ -53,6 +49,12 @@ public void handleTopBraidException() {
         // nothing to do
     }
 
+    @ResponseStatus(code = HttpStatus.SERVICE_UNAVAILABLE, reason = "Failed to connect to CVS")
+    @ExceptionHandler(CrosswalkServiceUnavailableException.class)
+    public void handleCrosswalkServiceUnavailableException() {
```

Comment:
you can make this a one liner if there's nothing to do 

`public void handleCrosswalkServiceUnavailableException() {}`

Reply:
and below as well